boards/shields/nrf700x_nrf54l15pdk/nrf700x_nrf54l15pdk.overlay

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <freq.h>
+
+/ {
+	/* Wi-Fi doesn't needs this and it frees up 68K SRAM */
+	/delete-node/ cpuflpr_sram;
+
+	chosen {
+		zephyr,wifi = &wlan0;
+	};
+};
+
+/* Adjust the cpuapp_sram to include the freed up cpuflpr_sram */
+&cpuapp_sram {
+	reg = <0x20000000 DT_SIZE_K(256)>;
+	ranges = <0x0 0x20000000 0x20040000>;
+};
+
+&pinctrl {
+	spi22_default: spi22_default {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 1, 11)>,
+				<NRF_PSEL(SPIM_MISO, 1, 9)>,
+				<NRF_PSEL(SPIM_MOSI, 1, 10)>;
+		};
+	};
+
+	spi22_sleep: spi22_sleep {
+		group1 {
+			psels = <NRF_PSEL(SPIM_SCK, 1, 11)>,
+				<NRF_PSEL(SPIM_MISO, 1, 9)>,
+				<NRF_PSEL(SPIM_MOSI, 1, 10)>;
+			low-power-enable;
+		};
+	};
+};
+
+/* With P1 only SPI20/21/22 are allowed and SPI20 uses same IRQ as UART20 */
+&spi22 {
```

Comment:
This is not a shield by definition. "Hardware shield-to-board compatibility depends on the use of well-known connectors used on popular boards (such as Arduino and 96boards). For software compatibility, boards must also provide a configuration matching their supported connectors." https://docs.zephyrproject.org/latest/hardware/porting/shields.html. This is a board-specific extension. Fine for downstream.

Reply:
yeah, I will make the nrf54l15pdk + the interposer board a board variant downstream, which can then be used with the nrf7002eb which is a proper shield with a generic connector (nordic edge connector used on Thingy kits)

Reply:
Right. I think it might be somewhat similar to https://docs.zephyrproject.org/latest/boards/shields/arduino_uno_click/doc/index.html where "The first socket (mikrobus_header_1) is the default socket which is assigned the node label mikrobus_header in the overlay." so building for arduino_uno_click + another_click_shield basically assumes that one has attached another_click_shield  to the "hard-coded" mikrobus_header_1. But I am not a big fan of the user experience it provides at the end of the day. 
So short answer might be: yeah, maybe just figure it out downstream, at least for now :)